src/collection/components/CollectionOrBundleEdit.tsx

```diff
@@ -324,6 +323,11 @@ const CollectionOrBundleEdit: FunctionComponent<
 		currentCollection: null,
 		initialCollection: null,
 	});
+	const isContributor = !!(collectionState.currentCollection?.contributors || []).find(
+		(contributor) => contributor.profile_id === user?.profile?.id
```

Comment:
Otherwise if both are undefined, the isContributor would be set to true
```suggestion
		(contributor) => !!contributor.profile_id && contributor.profile_id === user?.profile?.id
```